Source/WebCore/dom/Document.cpp

```diff
@@ -9625,6 +9627,8 @@ void Document::updateResizeObservations(Page& page)
     }
     if (!hasResizeObservers() && !m_resizeObserverForContainIntrinsicSize && !m_contentVisibilityDocumentState)
         return;
+    if (page.shouldDeferResizeEvents())
```

Comment:
"resize events" and "resize observations" are two separate things (the latter is ResizeObserver). This steps does not relate to resize events.